projects/plugins/jetpack/class.jetpack.php

```diff
@@ -4832,10 +4835,10 @@ public static function verify_onboarding_token( $token_data, $token, $request_da
 				&& isset( $jpo_token )
 				&& is_email( $jpo_user )
 				&& ctype_alnum( $jpo_token )
-				&& isset( $_GET['rest_route'] )
+				&& isset( $_GET['rest_route'] ) // phpcs:ignore WordPress.Security.NonceVerification.Recommended -- no site changes.
 				&& self::validate_onboarding_token_action(
 					$jpo_token,
-					$_GET['rest_route']
+					$_GET['rest_route'] // phpcs:ignore WordPress.Security.NonceVerification.Recommended, WordPress.Security.ValidatedSanitizedInput.MissingUnslash, WordPress.Security.ValidatedSanitizedInput.InputNotSanitized -- no site changes.
```

Comment:
Unslash should happen. InputNotSanitized is ok to ignore because validate_onboarding_token_action is doing the validation. Nonce is ok as above.

Reply:
4e8c9ca5db7b806f2f7b8e146ea95db47214fbe7